src/main/mw.c

```diff
@@ -267,7 +251,7 @@ void annexCode(void)
             DISABLE_ARMING_FLAG(OK_TO_ARM);
         }
 
-        if (isCalibrating()) {
+        if (isCalibrating() || (averageWaitingTasks100 > 100)) {
```

Comment:
`isCPUOverloaded()` ?

is there a better name for `averageWaitingTasks100`?  `averageTaskWaitingTasksPercent`? `cpuLoad`?